.travis.yml

```diff
@@ -52,7 +52,10 @@ matrix:
            env: JOB="Documentation" PYTHON_VERSION=3.5 SETUP_CMD='build_sphinx -w'
 
          - os: linux
-           env: JOB="Figures" PYTHON_VERSION=2.7 SETUP_CMD='test --figure' CONDA_DEPENDENCIES=''
+           env: JOB="Figures 2.7" PYTHON_VERSION=2.7 SETUP_CMD='test --figure' CONDA_DEPENDENCIES=''
```

Comment:
They only take ~5 minutes each, and it makes sure that people update both hash tables when an update is needed.

If time is a real issue, we could instead have `python setup.py test --figure` test *only* figures.